perftracker/views.py

```diff
@@ -54,6 +58,37 @@ def pt_redirect(request):
         return HttpResponseRedirect(redirect)
     return HttpResponse()
 
+
+@csrf_exempt
+def pt_auth(request, api_ver, project_id):
+    if request.method == 'GET':
+        data = {"is_authenticated": request.user.is_authenticated,
+                "username": str(request.user)}
+        return JsonResponse(data, safe=False)
+
+    elif request.method == 'POST':
+        if request.user.is_authenticated:
+            logout(request)
+
+        else:
+            try:
+                data = json.loads(request.body.decode("utf-8"))
+            except json.JSONDecodeError:
+                return HttpResponseBadRequest("Wrong json")
+
+            user = authenticate(request, username=data['email'], password=data['password'])
+            if user is not None:
+                login(request, user)
+            else:
+                logger.info("Authentication failed")
```

Comment:
log login name

Reply:
django-auth-ldap backend already logs all authentication attempts, so I don't need to do it manually. Moreover, I can even get rid of this 'custom' log.